biolink-model.yaml

```diff
@@ -9132,6 +9154,24 @@ classes:
         description: >-
           homology relationship type
 
+  gene to gene family association:
+    description: >-
+      Set membership of a gene in a family of genes related by common
+      evolutionary ancestry usually inferred by sequence comparisons.
+      The genes in a given family generally share common sequence motifs which
+      generally map onto shared gene product structure-function relationships.
+    is_a: association
+    defining_slots:
+      - subject
+      - predicate
+      - object
+    slot_usage:
```

Comment:
can we restrict the subject and object in this association too?

Reply:
Good  point! WIll do!